lib/rules/lines-around-comment.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @fileoverview Enforces empty lines around comments.
+ * @author Jamund Ferguson
+ * @copyright 2015 Jamund Ferguson. All rights reserved.
+ */
+"use strict";
+
+function getEmptyLines(context) {
```

Comment:
Missing JSDoc comment.
I think we should also work on the name, because this function doesn’t return the empty lines, it returns all line numbers of empty lines.